packages/validator-ajv8/jest.config.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
   coveragePathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/test'],
   coverageThreshold: {
     global: {
-      branches: 100,
+      // todo: dropped from 100 after jest 29
+      branches: 98,
```

Comment:
there is improvement in coverage reporting in new jest causing drop in branch coverage

Reply:
Please file a new issue to restore this back to 100%